common/browsers/firefox

```diff
@@ -13,7 +11,7 @@ if [[ -d $HOME/.mozilla/firefox ]]; then
             DIRArr[index]="$HOME/.mozilla/firefox/$profileItem"
         fi
         index=$index+1
-    done
+    done < <(grep '[P,p]'ath= $HOME/.mozilla/firefox/profiles.ini | sed 's/[P,p]ath=//')
```

Comment:
is the `,` really necessary? (guessing, it was for visibility, initially)
re https://github.com/graysky2/profile-sync-daemon/pull/224

Reply:
I think it could be even wrong. I simply copied it from the existing file because I was fixing something else.